tools/crm_error.c

```diff
@@ -86,6 +86,14 @@ main(int argc, char **argv)
         goto done;
     }
 
+    if (options.as_exit_code && options.as_rc) {
```

Comment:
The current lack of a check is a minor bug and no users should be specifying both options together -- but https://projects.clusterlabs.org/T261 makes me wonder if we need to wait on this.

Reply:
I'd like to think no one is relying on both of them combined being treated as `-X`. If they are, I would say it's a bug in **their** code...

And right, I think with text output, they're more likely to be checking the output than the return code anyway, so I don't know how much better an extra warning line in the output would be for them, compared to erroring and bailing out.

I don't have strong feelings and can change this to a warning if we want to.

Reply:
Yeah we get bit a lot when we assume no one's relying on something :)

Also, I'm not sure we want to reject this. A lot of OS commands accept contradictory arguments with the last one winning. Here that would require redoing the option values as an enum, but that's already in here later ...

Reply:
Last one winning would be fine and I can switch to that. The current behavior is not order-dependent -- if `-X` is present, we treat the value as an exit code no matter what: `-X`, `-rX`, `-Xr`. It's not intuitive that `-X` should automatically win.

Reply:
Yeah last one wins makes sense